lib/NP/Model/DnsRoot.pm

```diff
@@ -158,11 +158,35 @@ sub populate_country_zones {
         if (my $entries = $zone->active_servers('v6')) {
             @$entries = shuffle(@$entries);
 
-            # for now just put all IPv6 servers in the '2' zone
+            # add all servers to the non-numbered "NTP" zone
+            (my $pgeodns_group = "${name}") =~ s/\.$//;
+            push @{$data->{$name}->{aaaa}}, $_ for @$entries;
+
+            # Historically, only '2' has had ipv6 records, so we ALSO put all
+            # of the records there, for anyone who is relying on 2 having a
+            # large number of entries (Who would do such a crazy thing?)
```

Comment:
there's one of the "vendor zones" that needs this. The bits somewhere to specify the zone is an "sntp" or "ntp" zone is supposed to do something appropriate with this.

For regular ntp zones (the defaults?) we shouldn't duplicate between two numbered zones.